bokeh/model.py

```diff
@@ -264,16 +239,16 @@ def vm_serialize(self, changed_only=True):
 
         This method should be considered "private" or "protected",
         for use internal to Bokeh; use to_json() instead because
```

Comment:
Yeah - or get rid of it and put it right in to_json. I think the only complexity is there's some caller in models/ somewhere that I haven't really looked at. Maybe it can just use properties_with_values I'm not sure.